vagrant/shared.rb

```diff
@@ -1,12 +1,58 @@
 ENV['VAGRANT_NO_PARALLEL'] = 'yes'
 $ip = 2
 
+def get_username(base_box)
+	if base_box.start_with?('centos')
```

Comment:
`vagrant/shared.rb|11 warning| mismatched indentations at 'end' with 'if' at 5`

looks like a tab snuck in